### ISSUE TITLE
Display warning when classic editor is not available

### ISSUE THREAD

**Issue by jkva** [coyote-team/coyote-wp#115]
If Gutenberg cannot be properly integrated with, an admin notice should be displayed after the plugin is installed, communicating to the user that this plugin is incompatible with this editor.
